README.md

```diff
@@ -2,94 +2,60 @@
 
 <aside>
 
-👋 **Welcome!**
+We've put together a set of tools college instructors can use to easily prepare labs, assignments, or lectures. The content is self-designed so that it can be easily incorporated into the existing curriculum. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).
```

Comment:
nit: 
```suggestion
We've put together a set of tools college instructors can use to easily prepare labs, assignments, or lectures. The content is self-contained so that it can be easily incorporated into an existing curriculum. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).
```